src/Rule/Length.php

```diff
@@ -62,8 +71,12 @@ public function __construct($length)
      */
     public function validate($value)
     {
-        $actualLength = strlen($value);
-
+        if (is_null($this->encoding) || !function_exists('mb_strlen')) {
+            $actualLength = strlen($value);
+        } else {
+            $actualLength = mb_strlen($value, $this->encoding);
+        }
```

Comment:
Why not to use just `$actualLength = function_exists('mb_strlen') ? mb_strlen($value) : strlen($value)`?

I want to just use `->length(10)` with default encoding, I don't want to pass `'utf8'` everywhere.